release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -923,6 +923,18 @@ For information about using the {secrets-store-operator} to mount secrets from a
 
 For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file-nfs_persistent-storage-csi-azure-file[NFS support].
 
+[id="ocp-4-14-oci"]
+=== Oracle&#174; Cloud Infrastructure
+
+You can now install an {product-title} cluster on {oci} by using the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
```

Comment:
Should the full name be used in this first instance?
```suggestion
You can now install an {product-title} cluster on {oci-first} by using the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
```

Reply:
Good point, Updated.